s3.tf

```diff
@@ -1,3 +1,8 @@
+locals {
+  web_user_data_content    = sensitive(templatefile("${path.module}/templates/web_user_data.sh", local.web_interpolation_vars))
+  worker_user_data_content = sensitive(templatefile("${path.module}/templates/worker_user_data.sh", local.worker_interpolation_vars))
```

Comment:
Did you happen to double check that using sensitive here translates to the value being obfuscated when used in the terraforming of the s3 object below?